### ISSUE TITLE
API URL needs a fallback

### ISSUE THREAD

**Issue by Clorith** [WordPress/gutenberg#5758]
When creating a new site, WordPress now defaults to `/%postname%/` as the permalink structure, this means Gutenberg responds as if pretty permalinks were enabled, and uses the pretty API URL.

The problem occurs if there's no corresponding `.htaccess` file (or that file has an error in it), we could argue this is a core issue, but previously editing would not be affected by an incorrect permalink setting, so I feel like it should be addressed.

Perhaps it would be plausible with a fallback route, if the pretty permalink path fails, use the non-pretty links instead (we are getting a 404 not found, so that is very indicative of pretty permalinks not working. This might also help with those who block the JSON API for example, although they might not be using a 404 status, we could still check for failures in general I suspect).

**Comment by earnjam** [comment]
Gutenberg works without pretty permalinks enabled. In the event that `.htaccess` is missing or has an error, then the only thing that would work on the front-end of the site would be the homepage, so it doesn't really matter if they can edit content or not at that point. No one would be able to view it anyway. 

To me this more falls under #10492 where we could provide the user more information about why their posts aren't saving (REST API inaccessible). 

I don't think this necessarily needs to happen for 5.0 though.

**Comment by danielbachhuber** [comment]
> To me this more falls under #10492 where we could provide the user more information about why their posts aren't saving (REST API inaccessible).
> 
> I don't think this necessarily needs to happen for 5.0 though.

I agree with this assessment. Moving to `WordPress 5.0.x Follow Ups`.

**Comment by noisysocks** [comment]
[Had someone confused about this](https://wordpress.org/support/topic/posts-wont-save-or-publish/) and agree that some extra information about why the post can't be saved would be helpful.

**Comment by TimothyBJacobs** [comment]
This should no longer be an issue.
